### PR TITLE
Modified code in Sp_screen.kt

### PR DESCRIPTION
**Pull request by Tattaisreeram**

### CODE REVIEW

**Comment by ashut0sh75** [comment]
Hi @Tattaisreeram We are happy to have you and we really liked your enthusiasm to contribute in our project but we would like to guide you towards the rules for contributions in our readme file. This PR is under review process but in order to merge it please create an issue and it will be assigned to you only then this PR will be reviewed completely. Please go through the guidelines before making another contribution next time.

Thank you.

File: app/src/main/java/com/example/cartoon/Sp_screen.kt [hunk 1]
```diff
@@ -20,42 +18,40 @@ class Sp_Screen : AppCompatActivity() {
...
+    }
+
+    private fun startAnimation(
+        view: ImageView, property: String, startValue: Float, endValue: Float,
+        duration: Long, interpolator: DecelerateInterpolator
+    ) {
+        val animation = ObjectAnimator.ofFloat(view, property, startValue, endValue)
+        animation.duration = duration
+        animation.interpolator = interpolator
+        Handler().postDelayed({ animation.start() }, IMAGE_ANIMATION_DELAY)
```
Comment (ashut0sh75) [inline app/src/main/java/com/example/cartoon/Sp_screen.kt:47]: Handler().postDelayed({ animation.start() }, IMAGE_ANIMATION_DELAY)

Unresolved reference: IMAGE_ANIMATION_DELAY

File: app/src/main/java/com/example/cartoon/Sp_screen.kt [hunk 2]
```diff
@@ -20,42 +18,40 @@ class Sp_Screen : AppCompatActivity() {
...
+    }
+
+    private fun startAnimation(
+        view: TextView, property: String, startValue: Float, endValue: Float,
+        duration: Long, interpolator: AccelerateDecelerateInterpolator
+    ) {
+        val animation = ObjectAnimator.ofFloat(view, property, startValue, endValue)
+        animation.duration = duration
+        animation.interpolator = interpolator
+        Handler().postDelayed({ animation.start() }, TEXT_ANIMATION_DELAY)
```
Comment (ashut0sh75) [inline app/src/main/java/com/example/cartoon/Sp_screen.kt:57]: Handler().postDelayed({ animation.start() }, TEXT_ANIMATION_DELAY)

Unresolved reference: TEXT_ANIMATION_DELAY

**Review by ashut0sh75 — CHANGES_REQUESTED** [review]
The two lines are giving error.

File: app/src/main/java/com/example/cartoon/Sp_screen.kt (hunk 2 above)
Comment (Tattaisreeram) [reply app/src/main/java/com/example/cartoon/Sp_screen.kt:57]: Good to have a reply from you @ashut0sh75 I will resolve it and will give my contribution according to guidelines

**Comment by ashut0sh75** [comment]
Hi @Tattaisreeram please update your status on PR.

**Review by ashut0sh75 — APPROVED** [review]

**Comment by ashut0sh75** [comment]
No need to update status. I found the issues and resolved them.

**Pull request merged**